modules/lwjgl/core/src/generated/java/org/lwjgl/system/windows/Crypt32.java

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright LWJGL. All rights reserved.
+ * License terms: https://www.lwjgl.org/license
+ * MACHINE GENERATED FILE, DO NOT EDIT
+ */
+package org.lwjgl.system.windows;
+
+import javax.annotation.*;
+
+import java.nio.*;
+
+import org.lwjgl.*;
+
+import org.lwjgl.system.*;
+
+import static org.lwjgl.system.APIUtil.*;
+import static org.lwjgl.system.MemoryStack.*;
+import static org.lwjgl.system.MemoryUtil.*;
```

Comment:
This is missing an import for `org.lwjgl.system.Checks.*;` which seems to be a bug in the generator.

I assume this is caused by a discrepancy between 

https://github.com/LWJGL/lwjgl3/blob/ee2dc3bbdc8d601428c1108cf8486e2aaf18fd07/modules/generator/src/main/kotlin/org/lwjgl/generator/NativeClass.kt#L472-L485 and https://github.com/LWJGL/lwjgl3/blob/ee2dc3bbdc8d601428c1108cf8486e2aaf18fd07/modules/generator/src/main/kotlin/org/lwjgl/generator/Functions.kt#L471-L477 but I'm not sure how to best solve this.

Reply:
Thanks, fixed!